### ISSUE TITLE
Redo stats component

### ISSUE THREAD

**Issue by dmarcos** [aframevr/aframe#5397]
With something simpler more lightweight. Current is heavy and overkill.

**Comment by dmarcos** [comment]
Alternatives could be https://github.com/mrdoob/stats.js or https://github.com/RenaudRohlinger/stats-gl

**Comment by arpu** [comment]
https://github.com/TheoTheDev/three-perf

**Comment by dmarcos** [comment]
This shouldn't be too hard if someone wants to help

**Comment by arvind-iyer** [comment]
Is there a preference on which of these three to use? three-perf seems to be showing even more information than the existing stats UI, has anyone checked if it significantly faster?

**Comment by mrxz** [comment]
With WebGPU on the horizon, it's worth pointing out that `stats-gl` already supports it (despite its name).

Personally I don't think the `stats` component needs to show as much information as it currently does. The key point is showing the fps/raf and CPU/GPU timings. The other metrics can be nice, but have the potential downside of confusing users. It's not always trivial to interpret them, which can lead to people focussing on the wrong metrics when facing performance issues.

**Comment by dmarcos** [comment]
Yeah. stats should be much simpler. We could even have it to link to brief explanations on what of each numbers mean and low hanging fruit recommendations on how to improve them.
